client/src/components/new-request/analyze-results.vue

```diff
@@ -79,16 +79,19 @@
               </v-row>
 
             <!--GREY BOXES-->
-            <v-row class="row colour-p-blue-text no-gutters justify-center">
-              <v-col :key="issue.issue_type + '-' + option.header + '-' + optionIndex"
-                     cols="auto"
-                     v-for="(option, optionIndex) of issue.setup">
-                <GreyBox :i="optionIndex"
-                         :issueIndex="issueIndex"
-                         :option="option"
-                         :originalName="originalName" />
-              </v-col>
-            </v-row>
+
```

Comment:
Super minor but the comment line and spacing look a tad off.

Reply:
fixed